clean.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b     Delete bin directory"
+    echo "  -p     Delete packages directory and NuGet package caches"
+    echo "  -a     Remove all untracked files in the working tree and delete NuGet package caches"
```

Comment:
If this is deleting all the files produced by the build, the Nuget folder and cache, and cleans the working tree I would say that this is the clean all option.
A suggestion could be: "Delete all of the above and cleans the working tree" that way is also closer to the clean.cmd script